README.md

```diff
@@ -295,24 +295,25 @@ the GitHub release.
 
 The name for the ZIP must follow the following pattern:
 
-* `php_{extension-name}-{tag}-{php-maj/min}-{compiler}{"-nts"}?{-arch}.zip`
+* `php_{extension-name}-{tag}-{php-maj/min}-{ts|nts}-{compiler}-{arch}.zip`
+* Example: `php_xdebug-3.3.2-8.3-ts-vs16-x86_64.zip`
 
 The descriptions of these items:
 
 * `extension-name` the name of the extension, e.g. `xdebug`
 * `tag` for example `3.3.0alpha3` - defined by the tag/release you have made
 * `php-maj/min` - for example `8.3` for PHP 8.3.*
-* `compiler` - usually something like `vc16` - this should match
-  the `PHP_COMPILER_ID`
-* `-nts` - optional - non-thread safe. If omitted, thread-safe mode (ZTS) is
-  assumed.
-* `-arch` - for example `x86_64`, fetch using `php -r "echo php_uname('m');"`.
+* `compiler` - usually something like `vc6`, `vs16` - - fetch from
+  'PHP Extension Build' flags in `php -i`
+* `ts|nts` - Thread-safe or non-thread safe.
+* `arch` - for example `x86_64`, fetch using `php -r "echo php_uname('m');"` or
+  `Architecture` from `php -i`
```

Comment:
`php -i` doesn't have any word "Architecture" for me.

Reply:
`Architecture` is output on Windows builds only - if `PHP_BUILD_ARCH` is set, which is set in `toolset_setup_arch` to the value of `TARGET_ARCH`. That itself comes from `cl.exe`, it is parsed in `probe_binary` in `confutils.js`; matches `x86`, `x64`, and `ARM64` is replaced with `arm64`. This also explains why Windows currently has `x64` instead of `x86_64`.